.golangci.yml

```diff
@@ -13,8 +17,42 @@ linters-settings:
     enable=fieldalignment: true
   revive:
     rules:
-    - name: if-return
-      disabled: true
+      # The following rules are recommended https://github.com/mgechev/revive#recommended-configuration
```

Comment:
I think we merged the revive by mistake. 
Why should we have revive and golangci?

Also, note that we are only using golangci and we do not test the projects with revive.
In this way, ihmo we need to remove:

```
linters-settings:
  govet:
    enable=fieldalignment: true
  revive:
    rules:
    - name: if-return
      disabled: true
```

Reply:
To be honest I just noticed revive was enabled but not with the recommended configuration and therefor I made a pr to work towards the [recommended configuration](https://github.com/mgechev/revive#recommended-configuration). I believe that if you remove that section you will get [default configuration](https://github.com/mgechev/revive#default-configuration) instead.

As for the question of should there be / how much linting should a project have. I think that's a question you have more experience to answer. IMO there is a value for some linting. The question is always where do we get most value out of the least amount of linting overhead.

Reply:
HI @oscr,

> I believe that if you remove that section you will get [default configuration](https://github.com/mgechev/revive#default-configuration) instead.

I would we get that?
We are not downloading or testing using revive. We just use golangci.
Did I missed something?

Reply:
@camilamacedo86 golangci-lint is not a linter in itself, it's a linter runner. So you can configure golangci to run fewer or more or less linters. Then to make everything more complicated some linters (such as revive) are `metalinters` and they can be configured with even more checks. 

https://golangci-lint.run/usage/linters/